go.mod

```diff
@@ -27,6 +27,8 @@ replace github.com/davecgh/go-spew => github.com/davecgh/go-spew v1.1.0
 
 replace github.com/stretchr/testify => github.com/stretchr/testify v1.2.1
 
+replace github.com/grpc/grcp-go => github.com/grpc/grpc-go v1.33.1
```

Comment:
Did you add this manually? If so, why (OOC)?

Reply:
Oh good catch.  This was failing for me before you did your check-in for the release.  I think it's probably fine to remove it now, will do it if it checks out.